bin/docker/Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM       fedora:22
 
 # install the required dependencies to compile native extensions
 # and needed by the website to fetch resources
-RUN        dnf -y install gcc-c++ make ruby-devel libxml2-devel libxslt-devel findutils git ruby wget
+# also installs vim because who doesn't need it :) useful to edit Gemfiles and co
+RUN        dnf -y install gcc-c++ make ruby-devel libxml2-devel libxslt-devel findutils git ruby wget vim
```

Comment:
Not sure why but ```yum``` gets aborts when installing vim because of a conflict, sent a PR to remove it https://github.com/infinispan/infinispan.github.io/pull/35